docs/release-notes/next/PT-163191737-config-discovery-renaming.md

```diff
@@ -0,0 +1,5 @@
+* Changes user config discovery paths, i.e. the node is looking for the user config in:
+  - AETERNITY_CONFIG environment variable instead of EPOCH_CONFIG,
+  - ~/.aeternity/aeternity/aeternity.yaml file instead of ~/.epoch/epoch/epoch.yaml,
+  - ${AETERNITY_TOP}/aeternity.yaml file instead of ${AETERNITY_TOP}/epoch.yaml.
+  Backwards compatibility is kept for now, so user config defined in old locations will be working.
```

Comment:
until next major release

Reply:
@lucafavatella Can this be better worded either when preparing the release or in the separate PR? I'd rather not loose approvals by this change...